server/src/main/java/org/elasticsearch/snapshots/SnapshotShutdownProgressTracker.java

```diff
@@ -29,8 +29,12 @@
 import java.util.function.Supplier;
 
 /**
- * Tracks progress of shard snapshots during shutdown, on this single data node. Periodically reports progress via logging, the interval for
- * which see {@link #SNAPSHOT_PROGRESS_DURING_SHUTDOWN_LOG_INTERVAL_SETTING}.
+ * Tracks progress of shard snapshots during shutdown, on this single data node. Periodically reports progress via logging until
+ * all snapshots have completed or are paused, the interval for which see {@link #SNAPSHOT_PROGRESS_DURING_SHUTDOWN_LOG_INTERVAL_SETTING}.
+ * <P>
+ * Note that this class is used even when the node isn't shutting down. When {@link SnapshotShardsService} starts a new snapshot task,
```

Comment:
> Note that this class is used even when the node isn't shutting down.

Could you clarify that it always tracks snapshots but only logs about them when shutdown starts?